frontend/src/services/graphQLDevice.ts

```diff
@@ -67,6 +67,60 @@ const DEVICE_SELECT = `{
         }
       }
     }
+    events {
```

Comment:
Let's wait till we can merge in the Account linking branch, then break out the log query into its own model and request only on page load. We can use the new single device api.